docs/topics/hacking/debugging.rst

```diff
@@ -46,5 +46,38 @@ All being well it should look like this:
 .. image:: /screenshots/docker-ipdb.png
 
 
+Using the Django Debug Toolbar
+------------------------------
+
+The `Django Debug Toolbar`_ is very powerful and useful when viewing pages from
+the website, to check the view used, its parameters, the SQL queries, the
+templates rendered and their context...
+
+To enable it, create a `local_settings.py` file with the following content::
+
+    from settings import *  # noqa
+
+    INSTALLED_APPS += (
+        'debug_toolbar',
+    )
+    DEBUG_TOOLBAR_PATCH_SETTINGS = False  # Prevent DDT from patching the settings.
+
+    INTERNAL_IPS = ('127.0.0.1', '192.168.99.1')
```

Comment:
Well, the thing is, if we do it that way, then it's enabled for everything, including tests. And in that case, the test fails (when they do a `self.client.get` or the like).